datafusion/ffi/src/udf/mod.rs

```diff
@@ -66,15 +67,7 @@ pub struct FFI_ScalarUDF {
     /// FFI equivalent to the `volatility` of a [`ScalarUDF`]
     pub volatility: FFI_Volatility,
 
-    /// Determines the return type of the underlying [`ScalarUDF`] based on the
```

Comment:
I think this is a reduction in API surface now that we are mostly using Fields -- seems like a good change to me, but we might also want to call it out in the title/description of the PR